bananalyzer/__main__.py

```diff
@@ -322,14 +323,22 @@ def main() -> int:
         print("=======================================================================")
         return 0
 
-    for example in examples:
+    missing_mhtml = []
+    for i, example in enumerate(examples):
         if example.mhtml_url is not None:
             mhtml_path = get_examples_path() / example.id / "index.mhtml"
             if not mhtml_path.exists():
-                mhtml_str = download_mhtml(example.mhtml_url)
-                mhtml_path.parent.mkdir(parents=True, exist_ok=False)
-                with open(mhtml_path, "w") as file:
-                    file.write(mhtml_str)
+                try:
+                    mhtml_str = download_mhtml(example.mhtml_url)
+                except ClientError:
+                    missing_mhtml.append(i)
+                    continue
+                else:
+                    mhtml_path.parent.mkdir(parents=True, exist_ok=False)
+                    with open(mhtml_path, "w") as file:
+                        file.write(mhtml_str)
+
+    examples = [e for i, e in enumerate(examples) if i not in missing_mhtml]
```

Comment:
should just delete the id missing mhtml or handle it directly. Don't think we need handling for this (and could cause us to misunderstand how many tests we really have)